### DISCUSSION TITLE
Use DIANN with Command Line

### DISCUSSION THREAD

**Discussion by changweihewu** [vdemichev/DiaNN#643] [Q&A]
Dear Vadim, dear DIA-NN community,

        I'm running DIA-NN (1.8.1) in command line. result is good, but it have no a file name is “report.pdf”. I want to see some pictures by the file. I don’t know what parameter can help me gain that file.  if anyone can provide the solution, it would be greatly appreciated

Thanks,
Zhou

**Comment by vdemichev** [comment]
Hi Zhou,

The PDF report is generated by a separate binary that works only on Windows. The command to run it is printed by the GUI in the log at the end of the analysis.

Best,
Vadim

**Reply by changweihewu** [reply]
I see. Thank you!
